content/tutorial/tutorial.md

```diff
@@ -845,18 +845,6 @@ The Board component now looks like this:
 
 ```javascript{17,18}
 class Board extends React.Component {
-  handleClick(i) {
```

Comment:
should this replace "this.props.onClick" in renderSquaure?

Reply:
`this.props.onClick` can either be inside the handleClick method, or the `handleClick` method should also be deleted from the Board component since it isn't being called anywhere at that point so learners(like me) can follow along properly.